eng/ci/public-build.yml

```diff
@@ -13,13 +13,15 @@ pr:
     include:
     - v4.x
     - release_4.0
+    - feature/*
 
 trigger:
```

Comment:
These changes are fine for now, but in future it's cleaner to separate the CI changes from the code changes. It's easier to maintain and review and ensures changes like this don't get undone if we ever need to revert this PR for any reason.

Reply:
sounds good!